### PR TITLE
fix wrong log level on success

### LINKED ISSUE #3: successful user creation gets logged as error
## Steps to reproduce

1. Login as subadmin
2. Create a user
3. Login as admin
4. see logs: 

```
{"reqId":"NMGL2IZhLYFnOFygySBh","level":3,"time":"2019-06-28T09:05:23+00:00","remoteAddr":"192.168.56.1","user":"iz7LeRAmbR","app":"ldap_write_support","method":"POST","url":"/nextcloud/ocs/v2.php/cloud/users","message":"Create LDAP user 'p67eECLDiN' (cn=p67eecldin,dc=owncloud,dc=com","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:67.0) Gecko/20100101 Firefox/67.0","version":"16.0.1.1","id":"5d15d853dc5bd"}

[ldap_write_support] Error: Create LDAP user 'p67eECLDiN' (cn=p67eecldin,dc=owncloud,dc=com

POST /nextcloud/ocs/v2.php/cloud/users
from 192.168.56.1 by iz7LeRAmbR at 2019-06-28T09:05:23+00:00
```

## Test setup

https://github.com/nextcloud/server/wiki/How-to-test-LDAP%3F

## related code

https://github.com/nextcloud/ldap_write_support/blame/master/lib/LDAPUserManager.php#L237

### PR DESCRIPTION
**Pull request by blizzz**
fixes #3

### CODE REVIEW

File: lib/LDAPUserManager.php [hunk 1]
```diff
@@ -233,14 +233,19 @@ public function createUser($username, $password) {
 		$this->ensureAttribute($newUserEntry, $displayNameAttribute, $username);
 
 		$ret = ldap_add($connection, $newUserDN, $newUserEntry);
-		$message = $ret
-			? 'Create LDAP user \'{username}\' ({dn}'
-			: 'Unable to create LDAP user \'{username}\' ({dn})';
-		$this->logger->error($message, [
+
+		$message = 'Create LDAP user \'{username}\' ({dn}';
```
Comment (wiswedel) [inline lib/LDAPUserManager.php]: closing bracket missing

Comment (blizzz) [reply lib/LDAPUserManager.php]: good catch

Comment (blizzz) [reply lib/LDAPUserManager.php]: i let the CI run before pushing the typo fix. We have long queues today…

Comment (blizzz) [reply lib/LDAPUserManager.php]: green :)

**Pull request merged**